x/mint/keeper/querier_test.go

```diff
@@ -69,8 +89,8 @@ func TestQueryInflation(t *testing.T) {
 	require.Equal(t, app.MintKeeper.GetMinter(ctx).Inflation, inflation)
 }
 
-func TestQueryAnnualProvisions(t *testing.T) {
-	app, ctx := createTestApp(t, true)
+func (suite *MintKeeperTestSuite) TestQueryAnnualProvisions(t *testing.T) {
+	app, ctx := suite.app, suite.ctx
 	legacyQuerierCdc := codec.NewAminoCodec(app.LegacyAmino())
 	querier := keep.NewQuerier(app.MintKeeper, legacyQuerierCdc.LegacyAmino)
```

Comment:
Can you move this line to `SetupTest`? Looks common in all tests.

Reply:
Sure @aleem1314